Scripts/Runtime/WitRequestFactory.cs

```diff
@@ -30,12 +30,12 @@ public static WitRequest MessageRequest(this WitConfiguration config, string que
             List<WitRequest.QueryParam> queryParams = new List<WitRequest.QueryParam>();
             queryParams.Add(QueryParam("q", query));
 
-            if (-1 != requestOptions.nBestIntents)
+            if (null != requestOptions && -1 != requestOptions.nBestIntents)
```

Comment:
I thought request options was not nullable? Because we created a default one if one from the user wasn't provided?

Reply:
Nope, I hit an NPE with it. We'll have to do a bit more digging, but this will safeguard either way.